medicines/doc-index-updater/Cargo.toml

```diff
@@ -14,7 +14,7 @@ azure_sdk_service_bus = { git = "https://github.com/redbadger/AzureSDKForRust",
 azure_sdk_storage_blob = { git = "https://github.com/redbadger/AzureSDKForRust", branch = "service-bus-peek-lock-timeout" }
 azure_sdk_storage_core = { git = "https://github.com/redbadger/AzureSDKForRust", branch = "service-bus-peek-lock-timeout" }
 chrono = "0.4.11"
-base64 = "0.12.0"
+base64 = "0.11.0"
```

Comment:
Should we be moving backwards here?

Reply:
Unfortunately it was the choice between going back one version here, or updating the Azure SDK crate we're using. We've forked and branched that crate already, but we're close to getting that PR merged. Our use of this crate is small (one line outside this check) so it didn't seem worth upping that package for no real benefit.

Reply:
Fair enough.